linkcheck/htmlutil/linkparse.py

```diff
@@ -175,10 +175,10 @@ def start_element (self, tag, attrs, element_text, lineno, column):
         log.debug(LOG_CHECK, "line %d col %d", lineno, column)
         if tag == "base" and not self.base_ref:
             self.base_ref = attrs.get("href", u'')
-        ignoreclasses = ['broken_link','borked_link']
-        if tag == "a" and attrs.get('class') and ignoreclasses:
-            if any(item in ignoreclasses for item in attrs.get('class').split()):
-                log.debug(LOG_CHECK,"Found link classed \"%s\" to %s, not considering further", attrs.get('class'), attrs.get('href'))
+        # ignore_classes = ['broken_link', 'borked_link']
+        if tag == "a" and attrs.get('class') and ignore_classes:
+            if any(item in ignore_classes for item in attrs.get('class').split()):
+                log.debug(LOG_CHECK,'Found link classed "%s" to %s, not considering further', attrs.get('class'), attrs.get('href'))
```

Comment:
Spaces after commas, please

```suggestion
                log.debug(LOG_CHECK, 'Found link classed "%s" to %s, not considering further', attrs.get('class'), attrs.get('href'))
```